spec/helpers/cdn.js

```diff
@@ -209,4 +209,13 @@ describe('cdn helper', function () {
             },
         ], done);
     });
+
+    it('should handle case for img/icon-sprite.svg and remove / at the beggining of path before join', function (done) {
```

Comment:
Please change this description to something simpler and more generic, like `should not result in empty path segments`

Reply:
Also please squash your commits.